ui/src/index.js

```diff
@@ -24,7 +23,7 @@ sagaMiddleware.run(sagas);
 
 ReactDOM.render(
   <Provider store={store}>
-    <Router history={history}>
```

Comment:
it is not the root cause. See https://github.com/scality/metalk8s/pull/1140/commits/426692694934ef805dceb56a0206670e219ebd07